### ISSUE TITLE
more GUI options

### ISSUE THREAD

**Issue by szymonkaliski** [sdegutis/choose#4]
I think it would be great if we had more options with controlling colors in choose, now some of them are hardcoded, which blocks some ideas for customising, I think we should add options like this:

- [ ] set color of choose window (so we could create dark background one)
- [ ] set alpha channel of all colors, not just RGB value
- [ ] change icon next to input field (maybe just as a string, with UTF-8 there's lots of options to use different arrows)

**Comment by sdegutis** [comment]
I like this idea a million percent.

**Comment by daslastic** [comment]
yeah I want to have the option to make it AMOLED, the choose gui 👍🏻
also while I'm at it, thanks for your work, chipsenkbeil

**Comment by Lalit64** [comment]
@szymonkaliski Any updates on this?
It would also be great if we could:
Changing the rounding radius of the window, make it more flat so basically remove the macOS native window shadow, and a custom border color
